Feature/52 introduce topics (#63)
AbsaOSS/living-doc-generator: README.md

         |-- issue md page 1
         |-- issue md page 2
         |-- _index.md
      |-- _index.md
    |- org 2
      |--repo 1
         |-- issue md page 1
         |-- _index.md
      |--repo 2
          ...
      |-- _index.md
    |- _index.md
    ```

> ### Grouped Output by Issue Topics

Reviewer: ```suggestion
### Grouped Output by Topics
```

> The feature allows you to generate output grouped by issue topics. This feature is useful when you want to group issues by specific topics or themes.

Reviewer: ```suggestion
The feature allows you to generate output grouped by topics. This feature is useful when you want to group tickets by specific topics or themes.
```

Author: See commit: [f62aa4c](https://github.com/AbsaOSS/living-doc-generator/pull/63/commits/f62aa4ca4606f6e42dc9651590ca584e306e3f16)

To gain a better understanding of the term "Topic", refer to the [Labels](#labels) section.

- **Default Behavior**: By default, the action generates all the documentation in a single directory.

> - **Non-default Example**: Use the grouped output feature to organize the generated documentation by issue topics.

Reviewer: ```suggestion
- **Non-default Example**: Use the grouped output feature to organize the generated documentation by topics.
```

Author: Changes added: [f62aa4c](https://github.com/AbsaOSS/living-doc-generator/pull/63/commits/f62aa4ca4606f6e42dc9651590ca584e306e3f16).
